### PR TITLE
[vi] Unsupervised Learning

### PR DESCRIPTION
**Pull request by tt-anh-eole**

### CODE REVIEW

**Comment by tt-anh-eole** [comment]
Hi @shervinea , Could you add **review-wanted** label for me. Thanks.

**Comment by shervinea** [comment]
Thank you @tt-anh-eole for all your work, this is great! Let's start the review process.

File: vi/cs-229-unsupervised-learning.md [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+
+**18. Distortion function ― In order to see if the algorithm converges, we look at the distortion function defined as follows:**
+
+&#10230; Hàm Distortion - Để nhận biết khi nào thuật toán hội tụ, chúng ta sẽ xem xét hàm distortion được định nghĩa như sau: 
+
+<br>
+
+**19. Hierarchical clustering**
+
+&#10230; Hierarchical clustering
```
Comment (damminhtien) [inline vi/cs-229-unsupervised-learning.md:115]: I suggest using 'phân cụm phân cấp' or 'phân cụm theo thứ bậc'

**Comment by shervinea** [comment]
Thank you @tt-anh-eole and @damminhtien for all your help and work towards this translation! Proceeding with the merge now.

**Pull request merged**